parts/linux/cloud-init/artifacts/secure-tls-bootstrap.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=AKS Secure TLS Bootstrap Client
+ConditionPathExists=/usr/local/bin/aks-secure-tls-bootstrap-client
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+
+ExecStart=/usr/local/bin/aks-secure-tls-bootstrap-client \
```

Comment:
I'm guessing the binary is already present today ? it was brought in with another PR ?